src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -207,9 +208,9 @@ public void sendClosedDataSetElectionsMessage(List<Election> elections) throws M
             Map<String, List<Election>> reviewedDatasets = new HashMap<>();
             for(Election election: elections) {
                 List<Election> dsElections = electionDAO.findLastElectionsByReferenceIdAndType(election.getReferenceId(), ElectionType.DATA_SET.getValue());
-                DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(election.getReferenceId());
-                String dar_code = Objects.nonNull(dar) && Objects.nonNull(dar.getData()) ? dar.getData().getDarCode() : "";
-                reviewedDatasets.put(dar_code, dsElections);
+                DarCollection collection = collectionDAO.findDARCollectionByReferenceId(election.getReferenceId());
```

Comment:
This is another example of a dao call in a loop, but that's what we were doing before. Consider this optional. It would be nice to pull that call outside of the loop (as a list of collections) and then find the right one for the election being iterated over.